rakelib/docs.rake

```diff
@@ -64,6 +66,32 @@ begin
       packaged_modules
     ]
 
+    # Generate Puppet strings for built-in functions and data types.
+    # This task has no description so it does not appear in the task
+    # list.
+    desc ''
+    task :generate_strings do
+      @puppet_strings ||= begin
+        FileUtils.mkdir_p('tmp')
+        tmpfile = 'tmp/boltlib.json'
+
+        PuppetStrings.generate(
+          ['bolt-modules/*'],
+          markup: 'markdown',
+          json: true,
+          path: tmpfile,
+          yard_args: ['bolt-modules/boltlib',
+                      'bolt-modules/ctrl',
```

Comment:
Whoops, I think we missed `dir` the first time around here.

Reply:
:ohmy: Added